### ISSUE TITLE
error 403 always appear

### ISSUE THREAD

**Issue by Keezuzu** [etherealxx/volatile-concentration-localux-colab#40]
I already follow the instruction to fix it, but its not work, i already doing it for many attempt but it not work. Any more solutions?

**Comment by Keezuzu** [comment]
Nope, im not in icognito. Before this issue when erorr 403 solution it work, but somehow it not worked now (for me)

**Comment by etherealxx** [comment]
i need a way to replicate it. Would you mind if you share your current device OS and browser?

**Comment by Keezuzu** [comment]
Sure, Win 10 and Google Chrome
